servers/rendering/renderer_rd/shaders/tonemap.glsl

```diff
@@ -425,9 +426,10 @@ void main() {
 
 #ifndef SUBPASS
 	// Glow
-
 	if (params.use_glow && params.glow_mode != GLOW_MODE_MIX) {
 		vec3 glow = gather_glow(source_glow, uv_interp) * params.glow_intensity * params.luminance_multiplier;
+		if (params.glow_map_strength > 0.001)
+			glow = mix(glow, texture(glow_map, uv_interp).rgb * glow, params.glow_map_strength);
```

Comment:
As per the code style guidelines, braces are now required for single-line `if` statements:

```suggestion
		if (params.glow_map_strength > 0.001) {
			glow = mix(glow, texture(glow_map, uv_interp).rgb * glow, params.glow_map_strength);
		}
```

Reply:
I would have expected clang format to fix that. Good to know.